src/timetable/igd21.ts

```diff
@@ -1,6 +1,5 @@
 import {
-  BK_1,
-  BK_2,
+  BK,
```

Comment:
```suggestion
  BK_2,
```
I think you broke the imports. At least `BK_2` is still used. Did you tried to compile? `npm run build` (Install dependencies: `npm i`)